Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -148,11 +148,9 @@ def Simps.symm_apply (e : α ≃ β) : β → α := e.symm
 
 initialize_simps_projections Equiv (toFun → apply, invFun → symm_apply)
 
--- Porting note:
--- Added these lemmas as restatements of `left_inv` and `right_inv`,
--- which use the coercions.
--- We might even consider switching the names, and having these as a public API.
```

Comment:
This line of the comment might be worth keeping. But I don't particularly see that happening either.

Reply:
Yeah, I think let's just take an editorial decision against this TODO.